lib/ansible/module_utils/basic.py

```diff
@@ -1245,10 +1245,7 @@ def _check_locale(self):
             # fallback to the 'best' locale, per the function
             # final fallback is 'C', which may cause unicode issues
             # but is preferable to simply failing on unknown locale
-            try:
-                best_locale = get_best_parsable_locale(self)
-            except RuntimeError:
```

Comment:
I didn't realize get_best_parsable _locale was raising RuntimeWarning instead of RuntimeError, but I guess this was just a typo anyway.